templates.go

```diff
@@ -145,6 +148,9 @@ write_files:
                 # Set Felix logging to "info"
                 - name: FELIX_LOGSEVERITYSCREEN
                   value: "info"
+                # Set MTU for tunnel device used if ipip is enabled
+                - name: FELIX_IPINIPMTU
+                  value: "1440"
```

Comment:
Could this newly added MTU setting make our custom MTU setting in the config map obsolete? or do we need to set it to the same value? I just took the value they had in https://docs.projectcalico.org/v2.5/getting-started/kubernetes/installation/hosted/calico.yaml

Reply:
no, i would rather not screw with MTU, it can look like it's working and then way later we find some hidden problems